microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -3,254 +3,284 @@ description: The latest experimental features in Microsoft Edge DevTools
 title: Experimental features
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 03/15/2021 
+ms.date: 08/31/2021
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, experiment
 no-loc: ["Enable webhint", "Enable Network Console", "Source Order Viewer", "Enable Composited Layers in 3D View", "Enable new Font Editor tool within the Styles pane", "Enable new CSS Flexbox debugging features", "Enable + button tab menus to open more tools", "Enable Welcome tab", "3D View", "Turn on support to move tabs between panels", "Match keyboard shortcuts in the DevTools to Microsoft Visual Studio Code", "Edit keyboard shortcuts for any action in the DevTools", "Turn on new CSS grid debugging features", "Emulation: Support dual screen mode"]
```

Comment:
Question: I haden't seen this `no-loc` entry so far. Could you explain what this is about? Is it to flag terms that should not be translated in other languages?

Reply:
Thanks Michael, that makes sense. I'm a little surprised to be seeing entire sentences, and not just words. Example: "Match keyboard shortcuts in the DevTools to Microsoft Visual Studio Code". Perhaps this is because those appear in DevTools as is, and are not translated there either.